constants/icons.js

```diff
@@ -0,0 +1,40 @@
+import homeLogo from "../assets/homeLogo.png"
+import diamondBlue from "../assets/diamondBlue.png"
+import coinGold from "../assets/coinGold.png"
+import avatar from '../assets/avatar.png'
+import status from '../assets/statusOnline.png'
+import featuredQuest from '../assets/featuredQuest.png'
+import group from '../assets/Group.png'
+import cardHeader from '../assets/cardHeader.png'
+import blueIcon from '../assets/blueIcon.png'
+import character1 from '../assets/Character1.png'
+import character2 from '../assets/Character2.png'
+import character3 from '../assets/Character3.png'
+import character4 from '../assets/Character4.png'
+import character5 from '../assets/Character5.png'
+import character6 from '../assets/Character6.png'
+import homeLogoSmall from '../assets/homeLogoSmall.png'
+
+const ICONS = {
```

Comment:
There is no need for this variable to be defined, you can just import these assets in the components where you would need them

Reply:
I would like to keep it like this for this project. I will definitely do differently in next project onwards.